### ISSUE TITLE
Support for embedding Twitter threads

### ISSUE THREAD

**Issue by stoweboyd** [samwarnick/obsidian-simple-embeds#43]
The embedding of single tweets is great, but there doesn't seem to be a way to render an entire thread. Thoughts?

One option: support for embedding threads a la thread reader.

`[Twitter link](https://twitter.com/tannerc/thread/1539248244823404544)`

or with an attribute.

`[Twitter link|thread](https://twitter.com/tannerc/status/1539248240159334406)`

**Comment by samwarnick** [comment]
Yes, this would be great. Unfortunately, the first-party Twitter embed does not support showing entire threads. I haven't been able to find any alternative that could easily do this, but I'll keep my eyes open.
